framework/flask/__init__.py

```diff
@@ -19,6 +19,30 @@
 app.config['SENTRY_TAGS'] = {'App': 'web'}
 app.config['SENTRY_RELEASE'] = settings.VERSION
 
+def rm_handler(app, handler_name, func, key=None):
+    """Remove a handler from an application.
+    :param app: Flask app
+    :param handler_name: Name of handler type, e.g. 'before_request'
+    :param func: Handler function to attach
+    :param key: Blueprint name
+    """
+    handler_funcs_name = '{0}_funcs'.format(handler_name)
+    handler_funcs = getattr(app, handler_funcs_name)
+    try:
+        handler_funcs.get(key, []).remove(func)
+    except ValueError:
```

Comment:
@sloria I had to put this in because `rm_handlers` gets called repeatedly during unit tests and this exception makes all the tests fail.